nuntium/templates/nuntium/profiles/stats.html

```diff
@@ -0,0 +1,39 @@
+{% extends "base_edit.html" %}
+{% load i18n %}
+{% load pagination_tags %}
+{% load nuntium_tags %}
+{% load staticfiles %}
+
+{% block extrajs %}
```

Comment:
Yeah that's right but I'm wondering if to fix that here, I rather create a different issue =)